docs/guides/tracing.md

```diff
@@ -38,3 +40,26 @@ for the full list of environment variables you can configure.
 Note that you must specify one of `JAEGER_AGENT_HOST` or
 `JAEGER_SAMPLER_MANAGER_HOST_PORT` in each component for Jaeger to be enabled,
 even if you plan to use the default values.
+
+
+## OpenTelemetry
+
+### Dependencies
+
+In order to send traces you will need to set up an OpenTelemetry Collector. The collector will be able to send traces to
+multiple destinations such [AWS X-Ray](https://aws-otel.github.io/docs/getting-started/x-ray),
+[Google Cloud](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter),
+[DataDog](https://docs.datadoghq.com/tracing/trace_collection/open_standards/otel_collector_datadog_exporter/) and
+[others](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter). OpenTelemetry Collector
+provides a [helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-collector/examples/deployment-otlp-traces)
+to set up the environment.
+
+### Configuration
+
+The [Configuration file](https://cortexmetrics.io/docs/configuration/configuration-file/) has now a tracing section
+to configure the OpenTelemetry exporter.
+
+### Current State
+
+Cortex has not fully migrated from OpenTracing to OpenTelemetry and is currently using the
```

Comment:
Maybe mention that we are using the bridge to maintain backward compatibility with Jaeger, for now?